vllm/worker/cpu_model_runner.py

```diff
@@ -89,6 +94,24 @@ def load_model(self) -> None:
             scheduler_config=self.scheduler_config,
             cache_config=self.cache_config)
 
+        # Apply torch.compile to custom ops
+        from vllm.model_executor.custom_op import CustomOp
+
+        def replace_model(model: torch.nn.Module):
+            for _, m in model.named_children():
+                if isinstance(m, CustomOp):
+                    m.forward_native = torch.compile(m.forward_native,
+                                                     dynamic=True,
+                                                     options={
+                                                         "fx_graph_cache":
+                                                         True,
+                                                         "cpp_wrapper": True,
+                                                         "dce": True
+                                                     })
```

Comment:
Just wondering, can we do this in `CustomOp.forward_cpu` instead?

Reply:
I tried this, but it didn't work.
- The ```forward``` actually uses ```_forward_method```, so we should replace ```_forward_method```.
- If we replaced ```_forward_method```, the ```torch.complie``` will raise a error ```mutable rms_norm.default is not supported with cpp_wrapper```. Seems ```cpp_wrapper``` is not compatible with ```RMSNorm.forward_cuda```.

Reply:
I see. This doesn't look aesthetically good to me, but I don't have an alternative solution... 😞

Reply:
@youkaichao Could you please take a look at this part of code if you have time? Just wondering if you have any suggestion, as the code doesn't look ideal to me.